spec/webview-spec.coffee

```diff
@@ -142,6 +142,33 @@ describe '<webview> tag', ->
       webview.src = "data:text/html;base64,#{encoded}"
       document.body.appendChild webview
 
+  describe 'partition attribute', ->
+    isolatedWebview = null
+    beforeEach ->
+      isolatedWebview = new WebView
+
+    afterEach ->
+      document.body.removeChild isolatedWebview
+
+    it 'isolates storage for different id', (done) ->
+      listener = (e) ->
+        document.body.appendChild isolatedWebview
+        webview.removeEventListener 'did-finish-load', listener
+      listener2 = (e) ->
+        assert.equal e.message, "one 1"
+        webview.removeEventListener 'console-message', listener2
+      listener3 = (e) ->
+        assert.equal e.message, " 0"
+        isolatedWebview.removeEventListener 'console-message', listener3
+        done()
+      webview.addEventListener 'did-finish-load', listener
+      webview.addEventListener 'console-message', listener2
+      webview.src = "file://#{fixtures}/pages/partition/one.html"
```

Comment:
We don't have to write to localStorage in webview, we can just do it in current page since they share the same session. Also relying on the sequence of `console-message` make this test unstable since you didn't guarantee one page is loaded after the other.